surch/organization.py

```diff
@@ -21,9 +21,9 @@
 
 from . import logger, repo, utils, constants
 
+GITHUB_API_URL = 'https://api.github.com/{0}/{1}'
 REPO_DETAILS_API_URL = \
     'https://api.github.com/{0}/{1}/repos?type={2}&per_page={3}&page={4}'
```

Comment:
re-use GITHUB_API_URL by setting REPO_DETAILS_API_URL to GITHUB_API_URL+/repos?type={2}&per_page={3}&page={4}